code/modules/busy_space/organizations.dm

```diff
@@ -615,9 +859,47 @@
 	work = "courier and passenger transit"
 	headquarters = "Mars, Sol"
 	motto = "With Major Bill's, you won't pay major bills!"
+	annoying = 100 //oh god
 
-	ship_prefixes = list("TTV" = "transport", "TTV" = "luxury transit")
-	destination_names = list()
+	missions = list(
+		new /datum/lore/mission/prebuilt/transport("TTV"),
+		new /datum/lore/mission/prebuilt/freight("TFV"),
+		new /datum/lore/mission/prebuilt/luxury("TTV"),
+		)
+
+	serviced = list( // and THIS is basically just a population map of known space, slightly adjusted for distance
+		/datum/lore/system/vir = 5, //ppl mostly do not use major bills for in-system stuff, is my sense of it
+		/datum/lore/system/sol = 25,
+		/datum/lore/system/alpha_centauri = 20,
+		/datum/lore/system/tau_ceti = 20,
+		/datum/lore/system/kess_gendar = 15,
+		/datum/lore/system/new_seoul = 15,
+		/datum/lore/system/oasis = 15,//tourism pull
+		/datum/lore/system/kauqxum = 15,
+		/datum/lore/system/gavel = 10,
+		/datum/lore/system/saint_columbia = 10,
+		/datum/lore/system/mahimahi = 10,
+		/datum/lore/system/zhu_que = 10,
+		/datum/lore/system/love = 10,
+		/datum/lore/system/el = 10,
+		/datum/lore/system/abels_rest = 10,
+		/datum/lore/system/rarkajar = 5,
+		/datum/lore/system/eutopia = 5, //tourism pull
+		/datum/lore/system/raphael = 5,
+		/datum/lore/system/jahans_post = 5,
+		/datum/lore/system/relan = 5,
+		/datum/lore/system/exalts_light = 5,
+		/datum/lore/system/terminus = 3,
+		/datum/lore/system/phact = 3, //people really want to see the dumb castle
+		/datum/lore/system/qerrvalis = 3,
+		/datum/lore/system/nyx = 3,
+		/datum/lore/system/mesomori = 2,
+		/datum/lore/system/vounna = 2,
+		/datum/lore/system/arrathiir = 2,
+		/datum/lore/system/neon_light = 2,
```

Comment:
```suggestion
		/datum/lore/system/neon_light = 2
```
I changed my mind, Major Bills should go to Whythe. You know, for tourism!

Reply:
clearly we should run a metaplot about it